src/models/sparse.jl

```diff
@@ -57,3 +57,21 @@ function pullback_evaluate(∂B, tensor::SparseEquivTensor{T}, Rnl, Ylm,
    return ∂Rnl, ∂Ylm
 end
 
+
+_nl(bb) = [(n = b.n, l = b.l) for b in bb]
+
+function get_nl(tensor::SparseEquivTensor{T}) where {T}
+   # assume the new ACE model NEVER has the z channel
```

Comment:
yes, I think we want to guarantee *HERE* at least that there are only `(n, l, m)` channels. If that ever changes then we will of course have to revisit.